crates/interpreter/src/valid.rs

```diff
@@ -132,13 +132,18 @@ impl<'m> BranchTableApi<'m> for &mut Vec<BranchTableEntry> {
     }
 }
 
+struct MetadataView<'m> {
```

Comment:
This is good to have side table views specific to verification in `valid.rs` because `exec.rs` will need different views (it won't need the mutable index used only for verification).